definitions/variable/emissions/emissions.yaml

```
@@ -248,21 +284,24 @@
     unit: "{Level-3 Species}"
 
 - Emissions|{Level-2 Species}|Waste:
-    description: Emissions of {Level-2 Species} from waste (IPCC category 6)
-    unit: "{Level-2 Species}"
-
-- Emissions|{Level-2 Species}|Fossil Fuel Fires:
-    description: Emissions of {Level-2 Species} from fossil fuel fires (IPCC category 7)
+    description: Emissions of {Level-2 Species} from waste (IPCC 1996 category 6; IPCC 2006 category 4). 
+      Any emissions from fossil-based products (incineration or CO2 decomposition) should be accounted for here.
+      CO2 from organic waste handling and decay should not be included.
     unit: "{Level-2 Species}"
 
 - Emissions|{Level-2 Species}|Other:
-    description: Emissions of {Level-2 Species} from other sources
+    description: Emissions of {Level-2 Species} from other sources, (IPCC 1996 category 7; IPCC 2006 category 5B; including fossil fuel fires). 
+      Only use this category exceptionally, for any categories than cannot be accommodated in the categories described above. 
```

Why is "including fossil fuels" inside the brackets? And why should fossil fuel fires only be reported exceptionally?

Suggest to simplify.
```suggestion
    description: Emissions of {Level-2 Species} from other sources (IPCC 1996 category 7;
      IPCC 2006 category 5B) including fossil fuel fires and any sources or sinks that
      cannot be accommodated in other categories
```

---

Fossil fuel fires are reported in CEDS, and thus would have to go somewhere in a mapping to IAMC variables. It doesn't go anywhere else, but to me and Oliver it wasn't immediately clear where it should go. To remove confusion, I think it should be specified that we put it under "Other", not under e.g. "Energy". That's in line with EDGAR and CEDS.

---

Yes, I think it makes sense to put fossil fuel fires here, but then the rest of the description is strange - and I tried to clarify that with the alternative proposal...